drivers/wifi/nrf700x/osal/fw_if/umac_if/src/event.c

```diff
@@ -559,6 +561,110 @@ wifi_nrf_fmac_data_events_process(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 	return status;
 }
 
+#else /* CONFIG_NRF700X_RADIO_TEST */
+static enum wifi_nrf_status umac_event_rf_test_process(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
+						       void *event)
+{
+	enum wifi_nrf_status status = WIFI_NRF_STATUS_FAIL;
+	struct nrf_wifi_event_rftest *rf_test_event = NULL;
+	struct nrf_wifi_temperature_params rf_test_get_temperature;
+	struct nrf_wifi_rf_get_rf_rssi rf_get_rf_rssi;
+	struct nrf_wifi_rf_test_xo_calib xo_calib_params;
+	struct nrf_wifi_rf_get_xo_value rf_get_xo_value_params;
+
+	if (!event) {
+		wifi_nrf_osal_log_err(fmac_dev_ctx->fpriv->opriv,
+				      "%s: Invalid parameters\n",
+				      __func__);
+		goto out;
+	}
+
+	rf_test_event = ((struct nrf_wifi_event_rftest *)event);
+
+	if (rf_test_event->rf_test_info.rfevent[0] != fmac_dev_ctx->rf_test_type) {
+		wifi_nrf_osal_log_err(fmac_dev_ctx->fpriv->opriv,
+				      "%s: Invalid event type (%d) recd for RF test type (%d)\n",
+				      __func__,
+				      rf_test_event->rf_test_info.rfevent[0],
+				      fmac_dev_ctx->rf_test_type);
+		goto out;
+	}
+
+	switch (rf_test_event->rf_test_info.rfevent[0]) {
+	case NRF_WIFI_RF_TEST_EVENT_RX_ADC_CAP:
+	case NRF_WIFI_RF_TEST_EVENT_RX_STAT_PKT_CAP:
+	case NRF_WIFI_RF_TEST_EVENT_RX_DYN_PKT_CAP:
+		status = hal_rpu_mem_read(fmac_dev_ctx->hal_dev_ctx,
+					  fmac_dev_ctx->rf_test_cap_data,
+					  RPU_MEM_RF_TEST_CAP_BASE,
+					  fmac_dev_ctx->rf_test_cap_sz);
+
+		break;
+	case NRF_WIFI_RF_TEST_EVENT_TX_TONE_START:
+	case NRF_WIFI_RF_TEST_EVENT_DPD_ENABLE:
+		break;
+
+	case NRF_WIFI_RF_TEST_GET_TEMPERATURE:
+		memcpy(&rf_test_get_temperature,
+		(const unsigned char *)&rf_test_event->rf_test_info.rfevent[0],
```

Comment:
Indentation seems to be off. Please treat this as a generic comment for all further locations too.